src/components/DirectoryListItem.vue

```diff
@@ -28,6 +28,7 @@ export default {
 .image{
   width: $base-line-height * 4;
   height: $base-line-height * 4;
+  background-position: center center;
```

Comment:
I see the problem you are trying to solve here, but unfortunately, I don't think CSS will be the solution. By adding this line, you fixed some pictures but broke other pictures (which was the same conundrum we had). We believe the ultimate fix for this, is for staff/faculty/students to have more uniform/standard pictures taken. Until this comes to fruition, there won't be a good fix for this.